pkgs/development/tools/pyenv/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pyenv";
+  version = "2.3.21";
+
+  src = fetchFromGitHub {
+    owner = "pyenv";
+    repo = "pyenv";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-+rMkbrRSQBHX9qdifIpdLmQ8Uk39ott0xyJTD/mb+9w=";
+  };
+
+  configureScript = "src/configure";
```

Comment:
The configure script [has shebangs](https://github.com/pyenv/pyenv/blob/a2dff4809b8a7c387f26f6f5760c97959fdc940c/src/configure#L1) and cannot be built on Linux as is, so it is necessary to run patchShebangs.

Reply:
Thank you for the review and the help. This should be fixed now.